### PR TITLE
feat(argo-cd): Conditionally create the argocd-cmd-params-cm ConfigMap

### PR DESCRIPTION
**Pull request by joshuabaird**
This PR adds a new `configs.params.create` value that controls the creation of the `argocd-cmd-params-cm` _ConfigMap_ like the other _ConfigMap_ resources in this chart.  This is helpful for users who want to manage this _ConfigMap_ externally.

Checklist:

* [x] I have bumped the chart version according to [versioning](https://github.com/argoproj/argo-helm/blob/main/CONTRIBUTING.md#versioning)
* [x] I have updated the documentation according to [documentation](https://github.com/argoproj/argo-helm/blob/main/CONTRIBUTING.md#documentation)
* [x] I have updated the chart changelog with all the changes that come with this pull request according to [changelog](https://github.com/argoproj/argo-helm/blob/main/CONTRIBUTING.md#changelog).
* [x] Any new values are backwards compatible and/or have sensible default.
* [x] I have signed off all my commits as required by [DCO](https://github.com/argoproj/argoproj/blob/master/community/CONTRIBUTING.md).
* [x] My build is green ([troubleshooting builds](https://argo-cd.readthedocs.io/en/stable/developer-guide/ci/)).

### CODE REVIEW

File: charts/argo-cd/templates/argocd-configs/argocd-cmd-params-cm.yaml [hunk 1]
```diff
@@ -13,3 +14,4 @@ metadata:
   {{- end }}
 data:
   {{- include "argo-cd.config.params" . | trim | nindent 2 }}
+{{- end }}
```
Comment (yu-croco) [inline charts/argo-cd/templates/argocd-configs/argocd-cmd-params-cm.yaml:17]: nits; please add a new line in the end of file.

File: charts/argo-cd/Chart.yaml [hunk 2]
```diff
@@ -3,7 +3,7 @@ appVersion: v2.7.6
 kubeVersion: ">=1.23.0-0"
 description: A Helm chart for Argo CD, a declarative, GitOps continuous delivery tool for Kubernetes.
 name: argo-cd
-version: 5.36.6
+version: 5.36.8
```
Comment (yu-croco) [inline charts/argo-cd/Chart.yaml:6]: merge dependency? 🤔 
https://github.com/argoproj/argo-helm/pull/2127

Comment (joshuabaird) [reply charts/argo-cd/Chart.yaml:6]: Yeah - I didn't know how to handle that.  Can/should we bundle the two PR's in a single release?

Comment (yu-croco) [reply charts/argo-cd/Chart.yaml:6]: IMO
- It's better to separate release to reduce impact for users.
- We cannot promise which PR is merged ahead (this PR or your other PR or someone's PR), so I think this should be `5.36.7` for now

Comment (joshuabaird) [reply charts/argo-cd/Chart.yaml:6]: I did 5.36.8 for this one, because #2127 has already been approved and I figured it would be merged first.

Comment (yu-croco) [reply charts/argo-cd/Chart.yaml:6]: Though we need one more approve to merge, I'll leave it to you.

**Review by jmeridth — APPROVED** [review]

**Review by mbevc1 — APPROVED** [review]

**Pull request merged**